README.md

```diff
@@ -225,55 +156,29 @@ Example:
 
 > Describe 1 - 3 features you will add to the project.
```

Comment:
- [ ] To make this really customized, I think it is better to remove this generic suggestion provided by the template you used. Remember that users have to writing permission in your code, so there is nothing suggest in that regard. You can however replace with nice crafted statements about your proposed future changes. 🍷